bundles/io/org.eclipse.smarthome.io.rest.sitemap/src/main/java/org/eclipse/smarthome/io/rest/sitemap/SitemapSubscriptionService.java

```diff
@@ -50,11 +54,16 @@
  *
  * @author Kai Kreuzer - Initial contribution and API
  */
-@Component(service = SitemapSubscriptionService.class)
+@Component(service = SitemapSubscriptionService.class, configurationPid = "org.eclipse.smarthome.rest.sitemap", property = {
+        Constants.SERVICE_PID + "=org.eclipse.smarthome.rest.sitemap",
```

Comment:
All our configurable services use pids like `org.eclipse.smarthome.<servicename>`, so here we should use `org.eclipse.smarthome.sitemap`, if not even better `org.eclipse.smarthome.sitemapsubscription`.